pmlc/dialect/pxa/transforms/subgroups.cc

```diff
@@ -230,7 +232,9 @@ struct SubgroupsPass : public SubgroupsBase<SubgroupsPass> {
         6,       // Maximum register per thread
     };
     SubgroupCostModel cm(params, op);
-    if (cm.bestCost == std::numeric_limits<double>::infinity()) {
+    if (cm.bestCost == std::numeric_limits<double>::infinity() ||
+        op.getIVs().size() < 6) {
```

Comment:
Magic number 6, again.

Reply:
This is a pure hack to make sure we only 'subgroup' the convolution and not the eltwise stuff so I only have one sub-grouped thing to test and will definitely not make it into the non WIP commit.